### ISSUE TITLE
Allow setting the response_type in OAuth2WebServerFlow

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#201]
```
response_type is currently hard-coded to 'code':

http://code.google.com/p/google-api-python-client/source/browse/oauth2client/cli
ent.py#1072

It should default to 'code', but should be able to be changed.
```

Original issue reported on code.google.com by `jcgregorio@google.com` on 3 Oct 2012 at 6:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `jcgregorio@google.com` on 3 Oct 2012 at 7:27
- Added labels: **Component-oauth2client**, **Priority-High**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 8726ae3c3b1f.
```

Original comment by `jcgregorio@google.com` on 23 Oct 2012 at 8:14
- Changed state: **Fixed**
